app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/RefreshController.kt

```diff
@@ -414,17 +433,12 @@ class RefreshController @Inject constructor(
 
         apiResponse.data?.messages?.let { messages ->
 
-            var inboxUnreadCount: Int? = null
             write {
                 val upToDateFolder = getUpToDateFolder(folder.id)
                 val isConversationMode = localSettings.threadMode == ThreadMode.CONVERSATION
-                val allImpactedThreads = createThreads(scope, upToDateFolder, messages, isConversationMode).also { threads ->
-                    inboxUnreadCount = updateFoldersUnreadCount(
-                        foldersIds = (if (isConversationMode) threads.mapTo(mutableSetOf()) { it.folderId } else emptySet()) + folder.id,
-                        realm = this,
-                    )
-                }
+                val allImpactedThreads = createThreads(scope, upToDateFolder, messages, isConversationMode)
 
+                // TODO: This count will be false for INBOX & SNOOZED
```

Comment:
Maybe if we want to be clear?
```suggestion
                // TODO: This count will be false for INBOX & SNOOZED when the snooze feature will be implemented
```